integration/integration_test.go

```diff
@@ -186,6 +190,106 @@ func TestDownloadSyncthing(t *testing.T) {
 	}
 }
 
+func TestStacks(t *testing.T) {
+	if mode == "client" {
+		return
+	}
+
+	ctx := scopeagent.GetContextFromTest(t)
+	oktetoPath, err := getOktetoPath(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tName := fmt.Sprintf("TestStacks-%s", runtime.GOOS)
+	name := strings.ToLower(fmt.Sprintf("%s-%d", tName, time.Now().Unix()))
+	namespace := fmt.Sprintf("%s-%s", name, user)
+
+	test := scopeagent.GetTest(t)
+	test.Run(tName, func(t *testing.T) {
+		if err := createNamespace(ctx, oktetoPath, namespace); err != nil {
+			t.Fatal(err)
+		}
+
+		defer deleteNamespace(ctx, oktetoPath, namespace)
+
+		if err := cloneGitRepo(ctx, stackGitRepo); err != nil {
+			t.Fatal(err)
+		}
+		defer deleteGitRepo(ctx, stackGitFolder)
+		if err := deployStack(ctx, oktetoPath, stackManifest); err != nil {
+			t.Fatal(err)
+		}
+		endpoint := fmt.Sprintf("https://vote-%s.cloud.okteto.net", namespace)
+		content, err := getContent(endpoint)
```

Comment:
What about the rest of the services? I think we should also check that all the resources are created as expected as part of this test.